### ISSUE TITLE
django.conf.urls.patterns() is deprecated and will be removed in Django 1.10

### ISSUE THREAD

**Issue by haidaraM** [fabiocaccamo/django-maintenance-mode#7]
I had this warning when I installed the package. 

You can simply use a list of django.conf.urls.url() instances in `urls.py:
```python
urlpatterns = [
    url(r'^off/$', views.maintenance_mode_off, name='maintenance_mode_off'),
    url(r'^on/$', views.maintenance_mode_on, name='maintenance_mode_on'),
]
```

Full waring message : `RemovedInDjango110Warning: django.conf.urls.patterns() is deprecated and will be removed in Django 1.10. Update your urlpatterns to be a list of django.conf.urls.url() instances instead.
  url(r'^on/$', views.maintenance_mode_on, name='maintenance_mode_on'),`

**Comment by fabiocaccamo** [comment]
Could you make a PR?

**Comment by fabiocaccamo** [comment]
Fixed with 0.4.0 version

**Comment by haidaraM** [comment]
ok great
